tests/CocktailParty.tex

```diff
@@ -9,46 +9,45 @@ \subsection{Focus}
 \subsection{Setup}
 \begin{itemize}
 	\item \textbf{Party room}: any (large) room inside the apartment when normally a party would be held.
-	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing, some of them forming groups of 2 or 3 people, from which at least one is sitting. Three of the guests have drink orders assigned by the referees. The sitting person always has an assigned drink.
-	\item \textbf{Bar:} The bar is any flat surface where objects can be placed, in a room other than the \quotes{party room}. All available beverages are on top of the bar for the robot can see them.
-	\item \textbf{Barman:} The Barman bay be standing either behind the bar or next to it, depending on the arena setup.
+	\item \textbf{Guests:} At least five people are distributed in a predefined \quotes{party room} either sitting or standing.
+                Three of the guests have drink orders.
+	\item \textbf{Bar:} The bar is any flat surface where objects can be placed, in a room other than the \quotes{party room}.
+                All available beverages are on top of the bar.
+	\item \textbf{Bartender:} The Bartender may be standing either behind the bar or next to it, depending on the arena setup.
 \end{itemize}
 
 \subsection{Task}
 
 \begin{enumerate}
 
-	\item \textbf{Entering:} The robot enters the arena and navigates to the party room and waits for being called.
-
+	\item \textbf{Entering:} The robot enters the arena and navigates to the party room.
 	\item \textbf{Getting called:} The guests call the robot simultaneously, either rising an arm, waving, or shouting. The robot has to approach one of them.
-	% Remark: Themself is the correct word used instead of himself/herself  when person gender is unknown.
-	The calling person introduces themself by name before giving the order of a drink. The robot leads the dialogue to learn the person and retrieve their drink order. \\
-
-	The robot can decide to skip the detection of the calling and ask one person to walk in front of it. In this case, the referees determine the person to approach the robot.
+        Optionally, the robot can skip the call detection and ask for a person to step in front of it (the referees determine who approaches to the robot).
```

Comment:
Not all of the guests call the robot.

```\item \textbf{Getting called:} Some guests call the robot simultaneously, either rising an arm, waving, or shouting. The robot has to approach one of them.```

Reply:
What do you mean with "some"?

Reply:
If you say _the guests_, somehow most people will understand an implicit _all_ at the beginning of the sentence. However, we have at least 5 people and 3 orders, from which one is assigned to a sitting person (sitting people won't call the robot). So you have two people calling the robot at the same time.

A hard count would set the number of calling people to two. However the test must be designed for adaptability. Robots should be able to handle several calls, not only 2. Also, the 3 orders are only implicitly specified in the scoresheet. So to keep consistency, the test description i suggest just saying _some_, but you can leave the definite article should you find it non-ambiguous.